fw/http_tbl.c

```diff
@@ -439,10 +439,12 @@ tfw_cfgop_http_rule(TfwCfgSpec *cs, TfwCfgEntry *e)
 
 	val = tfw_http_val_adjust(in_field_val, field,
 				  &val_len, &type_val, &op_val);
-	if (IS_ERR(val))
+	if (IS_ERR(val)) {
+		kfree(arg);
 		return PTR_ERR(val);
+	}
 
-	rule = tfw_http_rule_new(tfw_chain_entry, type, arg_size);
+	rule = tfw_http_rule_new(tfw_chain_entry, type, arg_size);;
```

Comment:
extra `;`

Reply:
Doesn't checkpatch.pl reveal the typo? @EvgeniiMekhanik could you please check your git setup